exercises/C-comparison-operators/exercise.js

```diff
@@ -7,14 +7,14 @@
 
 var studentCount = 16;
 var mentorCount = 9;
-var moreStudentsThanMentors = true;
+var moreStudentsThanMentors = studentCount > mentorCount;
 
 var roomMaxCapacity = 25;
-var enoughSpaceInRoom = true;
+var enoughSpaceInRoom = roomMaxCapacity > (studentCount > mentorCount);
```

Comment:
Hi Askin.Great work. I only suggest this part because it may not be working properly for all other codes. Maybe you can consider below alternatives. Expect for this you are amazing.
var enoughSpaceInRoom = roomMaxCapacity >= studentCount + mentorCount;
var enoughSpaceInRoom = roomMaxCapacity >= 25;

Reply:
Many thanks for the feedback Adem. Yes, you are right, I made a small mistake while considering the goal of the variable. I have just modified the variable named enoughSpaceInRoom.